Server/login_screen.R

```diff
@@ -4,6 +4,11 @@
 # Date: July 13th, 2020
 # License: MIT License
 #####################################################################################################################
+observe({
```

Comment:
Can we change this `observe` by an `observeEvent(input$submit_details){...}`? I think `observeEvent` is slightly more efficient than `observe`.

Reply:
That, or maybe get rid of the login screen altogether?